include/decision_modes.h

```diff
@@ -0,0 +1,15 @@
+/**
+* @brief decision_modes definition.
+*
+* @file decision_modes.h
+* @author Chenxi Zhao
+* @date 2020-02-13
+*/
+#pragma once
+
+namespace reinforcement_learning {
+  enum decision_modes {

```

Comment:
`decision_modes` => `decision_mode` ?

Reply:
Not sure it is a pattern. For instance there is also [`queue_mode_enum`](https://github.com/VowpalWabbit/reinforcement_learning/blob/6f3ac9044bd0fdea5d148eb31bccd5101a5987dc/rlclientlib/logger/async_batcher.h#L17.

Reply:
Fixed.